src/cli.coffee

```diff
@@ -1,11 +1,5 @@
-asyncHandlers = require 'async-handlers'
-{docopt} = require 'docopt'
-fs = require 'fs-extra'
-path = require 'path'
 extensions = require './config/supported_file_extensions'
-
-
-options = docopt """
+usage = """
```

Comment:
This variable is used only once. You could keep the require of docopt here, and provide the usage string directly to it. I think that's still very readable.